packages/pymedphys_analysis/tests/tpscompare/test_dose_profiles.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (C) 2019 Cancer Care Associates
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version (the "AGPL-3.0+").
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License and the additional terms for more
+# details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# ADDITIONAL TERMS are also included as allowed by Section 7 of the GNU
+# Affero General Public License. These additional terms are Sections 1, 5,
+# 6, 7, 8, and 9 from the Apache License, Version 2.0 (the "Apache-2.0")
+# where all references to the definition "License" are instead defined to
+# mean the AGPL-3.0+.
+
+# You should have received a copy of the Apache-2.0 along with this
+# program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
+
+
+import os
+import json
+import lzma
+import operator
+
+import pytest
+
+import numpy as np
+import pandas as pd
+
+import pydicom
+
+from pymedphys_dicom.dicom import depth_dose, profile
+from pymedphys_analysis.tpscompare import bulk_load_mephysto
+
+from shared import (BASELINES_DIR, DATA_DIR,
+                    DICOM_DOSE_FILEPATHS, DICOM_PLAN_FILEPATH, DICOM_DIR,
+                    MEASUREMENTS_DIR)
+
+
+CREATE_BASELINE = False
+BASELINE_FILEPATH = os.path.join(BASELINES_DIR,
+                                 'dicom_dose_profiles.json')
+
+
+@pytest.fixture
+def loaded_doses():
+    doses = {}
+
+    for key, filepath in DICOM_DOSE_FILEPATHS.items():
+
+        with lzma.open(filepath) as a_file:
+            doses[key] = pydicom.dcmread(a_file, force=True)
+
+    return doses
+
+
+@pytest.fixture
+def loaded_plan():
+    plan = pydicom.read_file(str(DICOM_PLAN_FILEPATH), force=True)
+
+    return plan
+
+
+def test_bulk_compare(loaded_doses, loaded_plan):
+    absolute_dose_table = pd.read_csv(MEASUREMENTS_DIR.joinpath(
+        'AbsoluteDose.csv'), index_col=0)
+    absolute_dose = absolute_dose_table['d10 @ 90 SSD']['6 MV']
+
+    output_factors = pd.read_csv(
+        MEASUREMENTS_DIR.joinpath('OutputFactors.csv'), index_col=0)
+
+    absolute_doses = {
+        key: output_factors[key]['6 MV'] * absolute_dose
+        for key in output_factors.columns
+    }
+
+    absolute_scans_per_field = bulk_load_mephysto(
+        MEASUREMENTS_DIR, '06MV_(\d\dx\d\d)\.mcc', absolute_doses, 100)
+
+    getter = operator.itemgetter('displacement', 'dose')
+
+    for key, absolute_scans in absolute_scans_per_field.items():
+        dose_dataset = loaded_doses[key]
```

Comment:
And this is why!

Reply:
Sorry context? :)

Reply:
My bad haha, it was a follow on from [this comment](https://github.com/pymedphys/pymedphys/pull/427#discussion_r306145184).

The point basically being that a situation is likely to arise where you'll need to distinguish between a dose dataset and a dose grid, and the word "dose" is likely to be preferred for the latter.

Reply:
Yup, makes sense.